packages/amplify-graphql-api-construct-tests/src/__tests__/deploy-velocity/deploy-velocity-constants.ts

```diff
@@ -0,0 +1,81 @@
+import { AmplifyGraphqlApi } from '@aws-amplify/graphql-api-construct';
+
+const ONE_MINUTE = 60 * 1000;
+export const DURATION_10_MINUTES = 10 * ONE_MINUTE;
+export const DURATION_20_MINUTES = 20 * ONE_MINUTE;
+export const DURATION_30_MINUTES = 30 * ONE_MINUTE;
+
+export const COUNT_1_THOUSAND = 1000;
+export const COUNT_10_THOUSAND = 10000;
+export const COUNT_100_THOUSAND = 100000;
+
+export const SCHEMA_ONE_FIELD_NO_INDEX = /* GraphQL */ `
+  type Todo @model @auth(rules: [{ allow: public }]) {
+    field1: String!
+  }
+`;
+
+export const SCHEMA_ONE_FIELD_ALL_INDEXED = /* GraphQL */ `
+  type Todo @model @auth(rules: [{ allow: public }]) {
+    field1: String!
+  }
+`;
+
+export const SCHEMA_THREE_FIELDS_NO_INDEX = /* GraphQL */ `
+  type Todo @model @auth(rules: [{ allow: public }]) {
+    field1: String!
+    field2: String!
+    field3: String!
+  }
+`;
+
+export const SCHEMA_THREE_FIELDS_ALL_INDEXED = /* GraphQL */ `
```

Comment:
The schema is named with `ALL_INDEXED` but no `@index` found in the schema content. Same as the ones below.

Reply:
oh geez, yeah I missed that while copying things out into constants.